### DISCUSSION TITLE
🌟 StartStruck exchange

### DISCUSSION THREAD

**Discussion by montasim** [Schweinepriester/github-profile-achievements#456] [Q&A]
Repo: https://github.com/montasim/montasim

😊Thanks !

Provide any repo you want to receive Star in reply.

**Comment by d4tdev** [comment]
i have done
https://github.com/d4rtj/d4rtj

**Reply by montasim** [reply]
Done @d4rtj

**Reply by d4tdev** [reply]
excute me, can u create new discussions in your repo, i need more mark answer
https://github.com/STRINGLABITSOLUTIONS/Bhojon/discussions?discussions_q=is%3Aunanswered

**Reply by montasim** [reply]
@d4rtj New questions are added.

**Reply by fcasamento** [reply]
Done @dd4tj  StartStruck, could you repay me https://github.com/fcasamento/primeiro-terraform/tree/convert-azure

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by montasim** [reply]
@ArielMAJ done.

**Reply by ArielMAJ** [reply]
Ty

**Reply by d4tdev** [reply]
@ArielMAJ done

**Reply by ArielMAJ** [reply]
Ty @ttD4T

**Reply by bkpecho** [reply]
@ttD4T Done. Please star my repo too. ty

**REPO:** https://github.com/bkpecho/bkpecho

**Comment by ChrisHagedorn** [comment]
https://github.com/ChrisHagedorn/nextjs-with-wallet-adapter

Done! Could I get Galaxy Brain?

**Reply by fcasamento** [reply]
Done @ChrisHagedorn StartStruck, could you repay me https://github.com/fcasamento/primeiro-terraform/tree/convert-azure

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@ChrisHagedorn

**Comment by abhisheks008** [comment]
Star this repo, https://github.com/abhisheks008/ML-Crate

**Reply by montasim** [reply]
@abhisheks008 star my repository first, then I will star you back.

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by haithamaouati** [comment]
Done! Could I get Galaxy Brain?

**Reply by montasim** [reply]
@haithamaouati this discussion is for the StartStruck badge. Not for the Galaxy Brain badge.

**Comment by 1RB** [comment]
done 

here's mine:
* https://github.com/1RB/Authworria

**Reply by montasim** [reply]
Done @1RB

**Reply by ArielMAJ** [reply]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by muhammadnurulahsan** [comment]
star and ill star your repo back [link it below]
https://github.com/muhammadnurulahsan/Carpenco-Store-Client-Side
https://github.com/muhammadnurulahsan/Carpenco-Store-Server-Side

**Reply by montasim** [reply]
Done @muhammadnurulahsan

**Comment by fcasamento** [comment]
@montasim Done StartStruck, could you repay me https://github.com/fcasamento/primeiro-terraform/tree/convert-azure

**Reply by montasim** [reply]
Done @fcasamento

**Comment by ArielMAJ** [comment]
I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by LauraBigoni** [comment]
Please help me star my repo: https://github.com/LauraBigoni/WeatherApp
I will star you back, just leave the link to your repo too. Thank you guys

**Reply by ArielMAJ** [reply]
@LauraBigoni I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by LauraBigoni** [reply]
> @LauraBigoni I starred you, please star my repo too: https://github.com/ArielMAJ/test.me Feel free to contribute as well, there are many somewhat "simple" issues.

Did it, thank you very much

**Reply by bkpecho** [reply]
@LauraBigoni done! starred your repo. 

**Here's mine** https://github.com/bkpecho/bkpecho

**Comment by antoniohernan** [comment]
I starred you, please star my repo too: https://github.com/antoniohernan/putfiles2ec2
Feel free to contribute as well!!
Thank you guys

**Reply by LauraBigoni** [reply]
> I starred you, please star my repo too: https://github.com/antoniohernan/putfiles2ec2 Feel free to contribute as well!! Thank you guys
Thank you very much, i starred you back

**Reply by ArielMAJ** [reply]
Tyvm! I starred you back!

**Reply by bkpecho** [reply]
@antoniohernan done. Please star my repo too: https://github.com/bkpecho/bkpecho

Thank you!

**Reply by GameExplorer** [reply]
> I starred you, please star my repo too: https://github.com/antoniohernan/putfiles2ec2 Feel free to contribute as well!! Thank you guys

Starred your repo :) Here's mine: https://github.com/GameExplorer/GameExplorer

**Comment by GameExplorer** [comment]
https://github.com/GameExplorer/GameExplorer

Star 4 star

**Reply by ArielMAJ** [reply]
@GameExplorer I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by LauraBigoni** [reply]
> https://github.com/GameExplorer/GameExplorer
> 
> Star 4 star

starred your repo ^^ Please star mine back  https://github.com/antoniohernan/putfiles2ec2 🙇🏻‍♀️

**Reply by GameExplorer** [reply]
> > https://github.com/GameExplorer/GameExplorer
> > Star 4 star
> 
> starred your repo ^^ Please star mine back https://github.com/antoniohernan/putfiles2ec2 🙇🏻‍♀️

:+1:  Did it! :)

**Reply by bkpecho** [reply]
@GameExplorer hey I starred your repo. Here's mine 
**My repository:** https://github.com/bkpecho/bkpecho

**Reply by GameExplorer** [reply]
> @GameExplorer hey I starred your repo. Here's mine **My repository:** https://github.com/bkpecho/bkpecho

Hello, yeah starred yours too :)

**Reply by bkpecho** [reply]
Thanks!

**Comment by bkpecho** [comment]
@montasim done :)

I already starred ⭐ your repository. Feel free to star back mine :)
**My repository:** https://github.com/bkpecho/bkpecho

**Reply by montasim** [reply]
Done @bkpecho.

**Reply by bkpecho** [reply]
Thanks!

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Reply by ArielMAJ** [reply]
@4lfant I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by Rantoryu** [comment]
@montasim 
Done, please star my repo: https://github.com/Rantoryu/Personal_Website
I will star back anyone in the comment to this!

**Reply by montasim** [reply]
Done @Rantoryu
